backend/app/src/pong/entities/game.entities.ts

```diff
@@ -7,6 +7,21 @@ export interface HandshakeRequest extends Request {
   handshake?: { headers: { cookie: string } };
 }
 
+export enum UserConnectionStatus {
+  OFFLINE = 'OFFLINE',
+  ONLINE = 'ONLINE',
+  PLAYING = 'PLAYING',
+}
+
+export interface FrontendUser {
+  id: string;
+  avatarImg: string;
+  nickname: string;
+  eloScore: number;
+  twoFactorAuthenticationSet: boolean;
+  twoFactorAuthenticationSecret: string;
```

Comment:
We deleted the secret for security reason

Reply:
removed